Lib/test/test_long.py

```diff
@@ -1518,6 +1518,22 @@ def __init__(self, value):
         self.assertEqual(i, 1)
         self.assertEqual(getattr(i, 'foo', 'none'), 'bar')
 
+        class ValidBytes:
+            def __bytes__(self):
+                return b'\x01'
+        class InvalidBytes:
+            def __bytes__(self):
+                return 'abc'
+        class MissingBytes: ...
+        class RaisingBytes:
+            def __bytes__(self):
+                1 / 0
+
+        self.assertEqual(int.from_bytes(ValidBytes()), 1)
+        self.assertRaises(TypeError, int.from_bytes, InvalidBytes)
+        self.assertRaises(TypeError, int.from_bytes, MissingBytes)
+        self.assertRaises(TypeError, int.from_bytes, RaisingBytes)
```

Comment:
This shouldn't raise TypeError:

```
>>> class RaisingBytes:
...             def __bytes__(self):
...                 1 / 0
... 
>>> int.from_bytes(RaisingBytes())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<stdin>", line 3, in __bytes__
ZeroDivisionError: division by zero
```

And yet CI is green. Am I missing something? I'm compiling your PR locally now to see what's going on.

Reply:
Oh, Good catch! Even I missed this.